src/blocks/ErrorBoundary.css

```diff
@@ -0,0 +1,18 @@
+.error-message {
```

Comment:
Should we prefix this and other class anmes somehow to minimize the risk of affecting other existing elements on the page?